app/assets/css/sass/_popup.scss

```diff
@@ -536,6 +554,28 @@
 	white-space: nowrap;
 }
 
+li.rtmedia-list-item.media-type-music {
+	display: inline;
+
+	.rtmedia-item-title{
+		padding-left: 10px;
+	}
+}
+
+li.rtmedia-list-item.media-type-video {
+	display: inline;
+
+	.rtmedia-item-title{
+		padding-left: 10px;
+	}
+
+	.mejs-overlay-play{
+		height: 100% !important;
+		width: 100% !important;
```

Comment:
Check if we can remove the !important

Reply:
not its not possible , since im overwriting  the property which were given by js ( the js one cases issues with new ui)